src/analysis/retail/evoker/preservation/modules/talents/EmeraldCommunion.tsx

```diff
@@ -171,24 +170,7 @@ class EmeraldCommunion extends Analyzer {
                 ),
               });
             }
-            let ticksPerf = QualitativePerformance.Good;
-            const secondsChanneling = Math.max(0, (info.endChannelTime - info.timestamp) / 1000);
-            if (secondsChanneling < 3.5) {
-              ticksPerf = QualitativePerformance.Fail;
-            } else if (secondsChanneling < 4.5) {
-              ticksPerf = QualitativePerformance.Ok;
-            }
-            checklistItems.push({
-              label: (
-                <>
-                  Seconds of channeling{' '}
-                  <SpellLink spell={TALENTS_EVOKER.EMERALD_COMMUNION_TALENT} />
-                </>
-              ),
-              result: <PerformanceMark perf={ticksPerf} />,
-              details: <>{secondsChanneling.toFixed(1)} seconds</>,
-            });
-            const lowestPerf = getLowestPerf([ticksPerf, targetsHitPerf]);
+            const lowestPerf = getLowestPerf([targetsHitPerf]);
```

Comment:
The `getLowestPerf` call isn't necessary since there is only 1 perf, you can just use `targetsHitPerf` directly